tcl/rapidwright.tcl

```diff
@@ -53,7 +53,12 @@ proc compile_block_dcp { dcpFile } {
     puts "RAPIDWRIGHT_PATH=$rwpath"
     puts "CLASSPATH=$cpath"
     exec java com.xilinx.rapidwright.util.Unzip ${dcpFile} ${unzipDir}
-    read_xdc ${unzipDir}/${rootDcpFileName}_in_context.xdc
+    # Updated. Goal: avoid naming problems caused by the fact that the files copied into IP_CACHE have different names as the ones expected by RW. Error appears only in designs with multiple IPs with the same ID
+	# Line bellow removed and replaced by the following 2 lines
+    #read_xdc ${unzipDir}/${rootDcpFileName}_in_context.xdc
+	set file_name_xdc [glob -directory ${unzipDir} *_in_context.xdc]
```

Comment:
This looks like it should work.  I'm ok with this change, you can delete the old line as it stays in the history of git.